### ISSUE TITLE
Contributing Openmp Testcases in Fortran to OVO

### ISSUE THREAD

**Issue by seshakalyur** [TApplencourt/OvO#25]
Hello,

I would like to contribute Fortran testcases for validating various constructs in openmp 4.5 standards.
I had the following question:
Do you ONLY accept testcases with TARGET constructs that perform CPU/GPU offloading?
or
Do you accept testcases for other openmp constructs which do not necessarily perform offloading?

Thanks and Regards,
Sesha

**Comment by TApplencourt** [comment]
Hi Sesha,

Thanks for your inquiry! 

Originally OvO was developed to benchmark Offload. But I see no problem extending it if it follows the big OvO principle which differentiates it from other benchmarks. 

As you may know, OvO tests are generated. They test exhaustively all OpenMP constructs for parallelization / worksharing. I would like to keep this "meta-generation" aspect in OvO. 
Also because our tests are generated is not too hard now to generate tests for both Fortran and C++. I would like to keep these options if possible.

So in short If your tests are generated, I will be happy to work with you to get your tests merged.
If your tests are hand-written maybe https://github.com/SOLLVE/sollve_vv is the best place to start.

Hope this help,
Thomas
